test/scenarios/ynEIGEN/ynLSDeWithdrawals.t.sol

```diff
@@ -26,109 +26,36 @@ contract ynLSDeWithdrawalsTest is ynLSDeScenarioBaseTest {
     address public constant user = address(0x42069);
 
     ITokenStakingNode public tokenStakingNode;
-    RedemptionAssetsVault public redemptionAssetsVault;
-    WithdrawalQueueManager public withdrawalQueueManager;
-    LSDWrapper public wrapper;
 
     uint256 public constant AMOUNT = 1 ether;
 
     function setUp() public virtual override {
 
         super.setUp();
 
-        uint256 _totalAssetsBefore = yneigen.totalAssets();
-
         // deal assets to user
         {
             deal({ token: chainAddresses.lsd.WSTETH_ADDRESS, to: user, give: 1000 ether });
             deal({ token: chainAddresses.lsd.WOETH_ADDRESS, to: user, give: 1000 ether });
             deal({ token: chainAddresses.lsd.SFRXETH_ADDRESS, to: user, give: 1000 ether });
         }
 
-        // deploy RedemptionAssetsVault
```

Comment:
In terms of speed, deleting the setup() doesn't change much -they're just slow.

We need to write most tests for inner logic as unit tests or tests that don't interact directly with third party protocols.

And keep the tests that interact with eigenlayer contained - they still need to be done however

Whenever a foundry test touches an existing contract it's slow as it needs to access existing state. if it runs separately (eg. deposit into ynEigen)  it's fast.